travel_platform/Html/main.html

```diff
@@ -1,104 +1,111 @@
+  </div>
+</header>
 
-	<div class="row">
-	<div class="col-md-offset-4 col-md-8"><h2 class="text-center white">Common information</h2></div>
+<div class="row">
+  <div class="col-md-offset-4 col-md-8"><h2 class="text-center white">Common information</h2>
+  </div>
 
-	<div class="col-md-4 choose">
-		<a href="https://lviv.itea.ua/">
-			<img class="img-responsive" src="pics/planet.jpg" alt="planet">
-			<span class="text-center">Choose a city</span>
-		</a>
-	</div>
+  <div class="col-md-4 choose">
+    <a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/planet.jpg" alt="planet">
+      <span class="text-center">Choose a city</span>
+    </a>
+  </div>
 
-	<div class="col-md-8">
-		<div class="col-md-4 choose" id="museums"><a href="https://lviv.itea.ua/">
-		<img class="img-responsive" src="pics/museum.png" alt="museum">
-		<span class="text-center">Museums</span>
-		</a>
-		</div>
+  <div class="col-md-8">
+    <div class="col-md-4 choose" id="museums"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/museum.png" alt="museum">
+        <span class="text-center">Museums</span></a>
+    </div>
 
-		<div class="col-md-4 choose" id="attractions"><a href="https://lviv.itea.ua/">
-		<img class="img-responsive" src="pics/attraction.jpg" alt="attraction">
-		<span class="text-center">Attractions</span>
-		</a>
-		</div>
+    <div class="col-md-4 choose" id="attractions"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/attraction.jpg" alt="attraction">
+        <span class="text-center">Attractions</span></a>
+    </div>
 
-		<div class="col-md-4 choose" id="entertainment"><a href="https://lviv.itea.ua/">
-		<img class="img-responsive" src="pics/entertainment.jpg" alt="entertainment">
-		<span class="text-center">Entertainment</span>
-		</a>
-		</div>
+    <div class="col-md-4 choose" id="entertainment"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/entertainment.jpg" alt="entertainment">
+        <span class="text-center">Entertainment</span></a>
+    </div>
 
-		<div class="col-md-4 choose" id="hotels"><a href="https://lviv.itea.ua/">
-		<img class="img-responsive" src="pics/otel.jpg" alt="otel">
-		<span class="text-center">Hotels</span>
-		</a>
-		</div>
+    <div class="col-md-4 choose" id="hotels"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/otel.jpg" alt="otel">
+        <span class="text-center">Hotels</span></a>
+    </div>
 
-		<div class="col-md-4 choose" id="transport"><a href="https://lviv.itea.ua/">
-		<img class="img-responsive" src="pics/bus.jpg" alt="bus">
-		<span class="text-center">Transport</span>
-		</a>
-		</div>
+    <div class="col-md-4 choose" id="transport"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/bus.jpg" alt="bus">
+        <span class="text-center">Transport</span></a>
```

Comment:
Well, as I can see in my browser, it's not a part of image. I've changed text in `span` into "Transport HERE" and see the following picture:
![transport_span](https://cloud.githubusercontent.com/assets/15904354/26301060/a3d480b0-3ee8-11e7-80ea-d131a885a250.png)